ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-commands.sh

```diff
@@ -35,12 +35,15 @@ function pre-ocp-66839(){
         echo "Failed to extract manifests!"
         return 1
     fi
-    # There should not be any cap annotation in all extracted manifests 
-    curCap=$(grep -rh "capability.openshift.io/name:" "${manifestsDir}"|awk -F": " '{print $NF}'|sort -u)
-    if [[ "${curCap}" != "" ]]; then
-        echo "Caps in extracted manifests found: ${curCap}, but expected nothing"
+
+    # There should be only enabled cap annotaion in all extracted manifests
+    curCap=$(grep -rh "capability.openshift.io/name:" "${manifestsDir}"|awk -F": " '{print $NF}'|sort -u|xargs)
+    expectedCap=$(echo ${ADDITIONAL_ENABLED_CAPABILITIES} | tr ' ' '\n' | sort)
```

Comment:
`curCap` is a one-line string, `expectedCap` is a multi-lines string, right?

Reply:
```
# cat try.sh 
ADDITIONAL_ENABLED_CAPABILITIES="MachineAPI ImageRegistry"
test=$(echo ${ADDITIONAL_ENABLED_CAPABILITIES} | tr ' ' '\n' | sort)
echo ${test}
# ./try.sh 
ImageRegistry MachineAPI
```
Also one-line.

Reply:
```
$ cat test.sh 
ADDITIONAL_ENABLED_CAPABILITIES="MachineAPI ImageRegistry"
curCap=$(echo ${ADDITIONAL_ENABLED_CAPABILITIES} |awk -F": " '{print $NF}'|sort -u|xargs)
expectedCap=$(echo ${ADDITIONAL_ENABLED_CAPABILITIES} | tr ' ' '\n' | sort)

if [[ "${curCap}" != "${expectedCap}" ]]; then echo "failed"; fi

$ sh -x test.sh 
+ ADDITIONAL_ENABLED_CAPABILITIES='MachineAPI ImageRegistry'
++ echo MachineAPI ImageRegistry
++ awk '-F: ' '{print $NF}'
++ sort -u
++ xargs
+ curCap='MachineAPI ImageRegistry'
++ echo MachineAPI ImageRegistry
++ tr ' ' '\n'
++ sort
+ expectedCap='ImageRegistry
MachineAPI'
+ [[ MachineAPI ImageRegistry != \I\m\a\g\e\R\e\g\i\s\t\r\y\
\M\a\c\h\i\n\e\A\P\I ]]
+ echo failed
failed
```

If I changed `expectedCap=$(echo ${ADDITIONAL_ENABLED_CAPABILITIES} | tr ' ' '\n' | sort)` to `expectedCap=$(echo ${ADDITIONAL_ENABLED_CAPABILITIES} | sort -u | xargs)`, it works.

Reply:
You're right. we need xargs to change it to one-line. Let me update.